packages/amplify-cli/src/lib/pull-backend.js

```diff
@@ -0,0 +1,38 @@
+const { initializeEnv } = require('./initialize-env');
+const { constructInputParams, postPullCodeGenCheck } = require('./amplify-service-helper');
+
+async function pullBackend(context) {
+  context.exeInfo = context.amplify.getProjectDetails();
+  context.exeInfo.inputParams = constructInputParams(context);
+  context.print.info('');
+  context.print.info('Pre pull check:');
```

Comment:
Pre-pull?

Reply:
yes, the PM also pointed it out, and I've corrected it.